Jointify/Views/Subviews/Logo.swift

```diff
@@ -6,16 +6,22 @@
 //  Copyright © 2020 Lukas Gerhardt. All rights reserved.
 //
 
+// MARK: Imports
 import SwiftUI
 
+// MARK: - Logo
 struct Logo: View {
     
+    // MARK: Stored Type Properties
+    static let width: CGFloat = 192
+    static let height: CGFloat = 64
+    
     // MARK: Body
     var body: some View {
         Image("LogoMitText")
             .resizable()
             .aspectRatio(contentMode: .fill)
-            .frame(width: 192, height: 64)
+            .frame(width: Logo.width, height: Logo.height)
```

Comment:
I changed the values to Type Properties so I can access them for the height of the `BackButton`

Reply:
Great, I suggest definitely making it smaller (based on screenshots, I did not run it yet).

It seems to big to me compared to the logo. Might also be just a personal opinion though. :)